src/lib/EventPublisher.js

```diff
@@ -77,9 +77,9 @@ class EventPublisher extends EventEmitter {
 
 
   subscribe = (
-    eventName: ?string = ALL_EVENTS,
+    eventName: string = ALL_EVENTS,
```

Comment:
I think the issue is that default parameters have to come after required parameters. It would have to come after `eventHandler`.